src/util/configManager.js

```diff
@@ -0,0 +1,7 @@
+import ConfigManager from 'seng-config';
+import config from '../config/config';
+
+const configManager = new ConfigManager();
+configManager.init(config.config, config.environment);
+
+export default configManager;
```

Comment:
Why is this file located in the util folder and not in the config folder?

Reply:
It used to be created and registered in util/setupInjects.js, I’ve separated the ‘injectables’ into separate files but felt relocating/refactoring should be another issue.